src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -2290,15 +2288,106 @@ public void seekToBeginningAndReceive(TestContext context) {
         deleteAsync.await();
     }
 
+    @Test
+    public void notFoundToBeginningAndReceive(TestContext context) {
```

Comment:
notFound what? Can we give a meaningful name to this method? :-)

Reply:
Thanks, renamed